components/tft/tftspi.h

```diff
@@ -157,14 +157,21 @@
 #elif CONFIG_TFT_PREDEFINED_DISPLAY_TYPE == 5
 //CONFIG FOR TTGO T-DISPLAY
 #define DEFAULT_DISP_TYPE           DISP_TYPE_ST7789V
-#define DEFAULT_TFT_DISPLAY_WIDTH   240
-#define DEFAULT_TFT_DISPLAY_HEIGHT  320
+#define DEFAULT_TFT_DISPLAY_WIDTH   135
+#define DEFAULT_TFT_DISPLAY_HEIGHT  240
+
+//Need to be defined together so they can be swapped for x;y when rotating
+#define TFT_STATIC_WIDTH_OFFSET 53
+#define TFT_STATIC_HEIGHT_OFFSET 40
+
 #define DISP_COLOR_BITS_24          0x66
 #define DEFAULT_GAMMA_CURVE         0
 #define DEFAULT_SPI_CLOCK           20000000
 #define TFT_INVERT_ROTATION         0
 #define TFT_INVERT_ROTATION1        1
 #define TFT_RGB_BGR                 0x00
+//To be used by user application for initialization
+#define TFT_ALLWAYS_INVERTED
```

Comment:
How is this setting different from our other configurable inversion settings? Did you instead try changing TFT_INVERT_ROTATION / TFT_INVERT_ROTATION1 ? Also it is spelled wrong it should be "TFT_ALWAYS_INVERTED"

Reply:
It refers to the color inversion, also it is currently only there to be checked by the application, as i couldn't think of a reasonably clean way to do it in the library initialization function.